app/controllers/GuidanceController.scala

```diff
@@ -94,25 +97,27 @@ class GuidanceController @Inject() (
               case _ => Future.successful(BadRequest(errorHandler.badRequestTemplateWithProcessCode(Some(processCode))))
             }
           },
-          nextPageUrl => {
-            // nextPageUrl will become the value of the question or the input field data
-            // Using the page evaluation context submit the page, this will return None if the page needs to be redisplayed
-            // val pageContext = service.getPageContext(evalContext)
-            // BadRequest(questionView(page, pageContext, questionName(path)))
-            // Alternatively a Some(stanzaId) return should redirect to url = evalContext.stanzaIdMap(stanzaId).url
-            val redirectLocation  = routes.GuidanceController.getPage(processCode, nextPageUrl.url.drop(appConfig.baseUrl.length + processCode.length + 2))
-            service.submitPage(evalContext, s"/$path", nextPageUrl.url).map{
+          submittedAnswer => {
```

Comment:
IntelliiJ is reporting method exceeds cyclomatic complexity of 10. Perhaps the functionality for the submitted answer section could be moved to another method.